streampipes-extensions-management/src/main/java/org/apache/streampipes/extensions/management/connect/adapter/parser/util/JsonEventProperty.java

```diff
@@ -77,13 +79,15 @@ public static EventProperty getEventProperty(String key, Object o) {
 
       ((EventPropertyList) resultProperty).setEventProperty(arrayContent);
       resultProperty.setRuntimeName(key);
+
     }
 
     if (resultProperty == null) {
       LOG.error("Property Type was not detected in JsonParser for the schema detection. "
-          + "This should never happen!");
+                + "This should never happen!");
     }
 
+    resultProperty.setDescription("");
```

Comment:
Can't we retrieve the description from somewhere or could we pass it to the method?

Reply:
I am not quite sure. If I see it correctly the method `JsonEventProperty.getEventProperty` is mainly used in `GeoJsonParser`. There the description is not set.

Reply:
Alright, we can also try to investigate this in a/the follow-up issue